stl/inc/xstring

```diff
@@ -2288,6 +2291,12 @@ public:
         _Tidy_init();
     }
 
+    explicit basic_string(_SOCCC_allocator_arg, const _Alloc& _Al) noexcept

```

Comment:
As this is 2-argument, should we bother with `explicit`? (It makes a difference for `{arg1, arg2}` so it isn't completely ignored, but this is an internal constructor.)

Reply:
We can make `_SOCCC_allocator_arg` tag-like, that is, explicit default ctor (and maybe inline variable of that type) to prevent creating by mistake. Then string ctor should be non-explicit, because why yes